esquery.js

```diff
@@ -2,17 +2,17 @@
 import estraverse from 'estraverse';
 import parser from './parser.js';
 
-var isArray = Array.isArray;
+const {isArray} = Array;
```

Comment:
Could we enforce spacing inside destructuring and initialisers? Also, this variable is a leftover from a refactoring and we should probably just inline `Array.isArray` at its call sites.

Reply:
There is a rule to do so for all objects. Does this sound like what you want? https://eslint.org/docs/rules/object-curly-spacing

Reply:
I've inlined `Array.isArray`. I've also added a commit for enforcing `object-curly-spacing` rule (set to "always"), as it seemed to mostly match what was in the code base already. Let me know if that works for you.